addons/mail/static/tests/message/message_tests.js

```diff
@@ -672,6 +673,24 @@ QUnit.test("should not be able to reply to temporary/transient messages", async
     await contains(".o-mail-Message [title='Reply']", { count: 0 });
 });
 
+QUnit.test("squashed transient message should not have date in the sidebar", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "Channel 1", channel_type: "chat" });
+    const { openDiscuss } = await start();
+    openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "/who");
+    await click(".o-mail-Composer-send:enabled");
+    await contains(".o-mail-Message");
+
+    await insertText(".o-mail-Composer-input", "/who", { replace: true });
+    await click(".o-mail-Composer-send:enabled");
+    await click(".o-mail-Message.o-squashed");
+    await nextTick();
+    await contains(".o-mail-Message.o-squashed .o-mail-Message-sidebar .o-mail-Message-date", {
+        count: 0,
+    });
```

Comment:
Squashed message sidebar is shown on mouse hovering, not click

Reply:
I am not sure if there is any helper to activate hover state on it. So, i used click as it will also add a hover effect (there are some references of it where they have used click for hovering on message).